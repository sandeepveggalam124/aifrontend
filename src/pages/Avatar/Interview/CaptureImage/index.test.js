import { MemoryRouter as Router } from "react-router-dom";
import {render} from "@testing-library/react";
import CameraComponent from "./index";


it('renders correctly',()=>{
    const {container}=render(
        <Router>
            <CameraComponent/>
        </Router>
    )
    expect(container).toMatchSnapshot()
})