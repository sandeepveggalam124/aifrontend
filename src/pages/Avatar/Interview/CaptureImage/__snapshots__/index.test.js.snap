// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    class="embed-container d-flex flex-column align-items-center justify-content-center img-fluid"
  >
    <video
      autoplay=""
      class="responsive-webcam"
      playsinline=""
    />
  </div>
</div>
`;
